descriptors/ModuleDescriptor-template.json

```diff
@@ -482,6 +482,16 @@
             "circulation.check-in-by-barcode.post"
           ]
         },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/inn-reach/transactions/{id}/itemhold/finalcheckin/{servicePointId}",
+          "permissionsRequired": ["inn-reach.d2ir.inn-reach-transaction.item.post"],
+          "modulePermissions": [
+            "circulation.requests.item.get",
```

Comment:
Do we need a permission to get a request for this endpoint?

Reply:
removed